dhis-2/dhis-support/dhis-support-hibernate/src/main/resources/hibernate-default.properties

```diff
@@ -13,7 +13,7 @@
 # hibernate.dialect = org.hibernate.dialect.H2Dialect
 hibernate.dialect = org.hisp.dhis.hibernate.dialect.DhisH2Dialect
 hibernate.connection.driver_class = org.h2.Driver
-hibernate.connection.url = jdbc:h2:mem:dhis2;DB_CLOSE_DELAY=-1;MVCC=TRUE;ALIAS_COLUMN_NAME=TRUE;DB_CLOSE_ON_EXIT=FALSE
+hibernate.connection.url = jdbc:h2:mem:dhis2;DB_CLOSE_DELAY=-1;MVCC=TRUE;ALIAS_COLUMN_NAME=TRUE;DB_CLOSE_ON_EXIT=FALSE;INIT=create domain if not exists jsonb as other;
 hibernate.connection.username = sa

```

Comment:
What does this line do?

Reply:
This line is needed so that H2 knows how to handle jsonb columns. Otherwise, when running InterpretationServiceTest you get an error for the methods that have to deal with the mentions column

Reply:
OK thanks.